docusaurus/docs/gers/gers.mdx

```diff
@@ -5,22 +5,20 @@ label: GERS
 
 Overture's Global Entity Reference System (GERS) is a framework for structuring, encoding and referencing map data to a shared universal reference. Overture is building this common ID system to solve the difficult problem of integrating geospatial datasets; to make it easier to share and sell map data; and to enable cooperation among companies, non-profits, government agencies and other organizations.
 
-## Defining an entity
+GERS provides a mechanism to conflate datasets, matching one or more entities via a unique GERS ID. For example, two geospatial datasets each containing a polygon that represents the Empire State Building can be easily conflated because both polygons will contain the same GERS ID, referring to the entity defined as: "a polygonal representation of the footprint of New York's Empire State Building."
 
-GERS provides a mechanism to conflate datasets, matching _entity_ to _entity_ via a unique GERS ID. An _entity_ is basically a thing in the real world. That thing could be a segment of road, a city, a grocery store, a building, a park bench or a lamp post. An _entity_ in GERS is assigned a unique GERS ID. These IDs are useful to anyone looking to match their data with Overture data.
 
-For example, two geospatial datasets each containing a polygon that represents the Empire State Building can be easily conflated because both polygons will contain the same GERS ID, referring to the _entity_ defined as: "a polygonal representation of the footprint of New York's Empire State Building."
+## GERS ID
 
-
-## Components of a GERS ID
-
-The Global Entity Reference System generates IDs in a consistent format:
+Overture generates a unique ID for an entity the first time that entity interacts with GERS. GERS IDs have the following structure:
 
 `H3 tile address` +
 `data theme identifier` +
-`56-bit value applied uniformly across a given theme`
+`56-bit value dependent on theme`
 
-Here you can see a list of IDs from Overture's recent release of buildings data, pulled from the GeoParquet file with a simple [duckdb](https://duckdb.org/) query. You can find more information on accessing and querying Overture datasets in the How-to section of our documentation.
+The H3 index reflects the location of an entity when Overture first assigns it an ID. Because GERS IDs are intended to be stable over time, Overture will not generate a new ID for that entity if its location changes slightly. The same is true for the data theme identifier; if Overture moves an entity from an existing data theme to a new data theme, the ID will not change. Put another way, a GERS ID should be treated as a unique identifier and nothing more. Overture discourages users from attempting to extract information about an entity from its GERS ID.
```

Comment:
Let's scrap this part: ~The same is true for the data theme identifier; if Overture moves an entity from an existing data theme to a new data theme, the ID will not change~ — in that case the ID would likely change